src/main/java/org/swisspush/redisques/QueueStatsService.java

```diff
@@ -50,15 +48,17 @@ public class QueueStatsService {
     private final DequeueStatisticCollector dequeueStatisticCollector;
     private final RedisQuesExceptionFactory exceptionFactory;
     private final Semaphore incomingRequestQuota;
+    private final Boolean  fetchQueueStats;
 
     public QueueStatsService(
-        Vertx vertx,
-        EventBus eventBus,
-        String redisquesAddress,
-        QueueStatisticsCollector queueStatisticsCollector,
-        DequeueStatisticCollector dequeueStatisticCollector,
-        RedisQuesExceptionFactory exceptionFactory,
-        Semaphore incomingRequestQuota
+            Vertx vertx,
+            EventBus eventBus,
+            String redisquesAddress,
+            QueueStatisticsCollector queueStatisticsCollector,
+            DequeueStatisticCollector dequeueStatisticCollector,
+            RedisQuesExceptionFactory exceptionFactory,
+            Semaphore incomingRequestQuota,
+            Boolean fetchQueueStats
```

Comment:
Use `boolean` instead of `Boolean`. Why is the parameter called `fetchQueueStats` when the method it is used for is called `attachDequeueStats`?

I suggest calling it `enableDequeueStats` or something similar

Reply:
I also think enableDequeueStats is more suitable, I will implement.